spec/system/custom_dropdown_field_user_field_spec.rb

```diff
@@ -18,7 +16,7 @@
       has_custom_validation: false,
       show_values: [],
       target_user_field_ids: [],
-    ) { user_field_options { [Fabricate(:user_field_option, value: NOT_A_SHOW_VALIDATION_VALUE)] } }
+    ) { user_field_options { [Fabricate(:user_field_option, value: "not a show_values value")] } }
```

Comment:
There is no way to load the `value` from the page object here... sadly we have to hardcode the value string instead.